Windows (#9468)
fishtown-analytics/docs.getdbt.com: website/docs/guides/fusion-qs.md

2. Search for `dbt` and choose the one from the publisher `dbt Labs, LLC`.
    <Lightbox src="/img/docs/extension/extension-marketplace.png" width="60%" title="Search for the extension"/>
3. Click **Install**.

> 4. When the prompt appears, you can register the extension now or skip it (you can register later). You can also check out [Sign in or register](/docs/sign-in-dbt-extension) or the [installation instructions](/docs/install-dbt-extension) to come back to it later.

Reviewer: ```suggestion
4. When the prompt appears, you can register the extension or skip it. Check out the [registration](/docs/sign-in-dbt-extension) or [installation](/docs/install-dbt-extension) instructions to come back to it later.
```
